examples/android/android-studio/Cukeulator/README.md

```diff
@@ -0,0 +1,60 @@
+## Cukeulator Example Test
+This is the example test-project for the Cukeulator app for Android Studio (Beta) 0.8.14.
+
+### Setup
+Features must be placed in `assets/features/`. Subdirectories are allowed.
+
+The `app/libs` should contain `cucumber-android-*.jar`. Please build it following [cucumber-jvm/android]
+(https://github.com/cucumber/cucumber-jvm/tree/master/android) instructions and copy to `app/libs` folder.
+
+The rest of the dependencies are added automatically in `app/build.gradle`.
+
+
+### Using gradle
+
+To build the test apk:
```

Comment:
This works also. I will update the readme.

Thank you!

- Mykola

On Sat, Jan 3, 2015 at 7:19 AM, Sebastian Gröbler <notifications@github.com>
wrote:

> In examples/android/android-studio/Cukeulator/README.md
> <https://github.com/cucumber/cucumber-jvm/pull/816#discussion-diff-22432101>
> :
>
> > @@ -0,0 +1,60 @@
> > +## Cukeulator Example Test
> > +This is the example test-project for the Cukeulator app for Android Studio (Beta) 0.8.14.
> > +
> > +### Setup
> > +Features must be placed in `assets/features/`. Subdirectories are allowed.
> > +
> > +The `app/libs` should contain `cucumber-android-*.jar`. Please build it following [cucumber-jvm/android]
> > +(https://github.com/cucumber/cucumber-jvm/tree/master/android) instructions and copy to `app/libs` folder.
> > +
> > +The rest of the dependencies are added automatically in `app/build.gradle`.
> > +
> > +
> > +### Using gradle
> > +
> > +To build the test apk:
>
> in case all the following steps are just about running the tests, why not
> simply say to run ./gradlew connectedCheck?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/cucumber/cucumber-jvm/pull/816/files#r22432101>.
>